docs/man/xrdp-sesman.8.in

```diff
@@ -34,6 +34,10 @@ Output version information and exit.
 \fB\-n\fR, \fB\-\-nodaemon\fR
 Starts \fBxrdp\-sesman\fR in foreground instead of starting it as a daemon.
 .TP
+\fB\-\-log\-config\fR
+Print the configuration on stdout before starting the daemon.
+The default is not to do this.
+.TP
```

Comment:
Thinking outside the context of this pull request, when I think of the command line argument `--log-config` I'd assume that this argument is for specifying where xrdp should read the logging configuration (eg. File name, or inline config definition). This is reinforced by the similarly named `--config` argument which does specify where to read the config from.

Alternatively, the command line argument `--log-config` also made me think that it means "print all of the config (port, security settings, keyboard layout, ...) to the log" which is a nice feature to have but is not the stated goal of this pull request.

I suggest changing the command line argument name to make the purpose clearer. For example: `--print-logging-config` or `--dump-log-config`

Reply:
Thanks - I'll do that.

Reply:
I've gone for `--dump-config` in the end, as for xrdp-sesman the variable is also used to control whether the main part of sesman.ini is also dumped.